app/views/how-tos/updating-the-kit.html

```diff
@@ -20,7 +20,7 @@ <h1>
       </h1>
 
       {{ insetText({
-        html: "<p>If you are upgrading to version 7.0.0 of the NHS prototype kit, this also upgrades NHS frontend to version 10.0.0. You will need to follow the <a href=\"https://service-manual.nhs.uk/design-system/guides/updating-to-v10\">upgrading to version 10 guide</a> to make any required changes to your code.</p>"
+        html: "<p>If you are upgrading to version 7+ of the NHS prototype kit, this also upgrades NHS frontend to version 10+ You will need to follow the <a href=\"https://service-manual.nhs.uk/design-system/guides/updating-to-v10\">upgrading to version 10 guide</a> to make any required changes to your code.</p>"
```

Comment:
Maybe this is clearer?

```suggestion
        html: "<p>If you are upgrading to version 7.x of the NHS prototype kit, this also upgrades NHS frontend to version 10.x You will need to follow the <a href=\"https://service-manual.nhs.uk/design-system/guides/updating-to-v10\">upgrading to version 10 guide</a> to make any required changes to your code.</p>"
```

Reply:
@colinrotherham how's this?

```suggestion
        html: "<p>If you are upgrading to NHS prototype kit version 7+, this also upgrades NHS frontend to version 10+ You will need to follow the <a href=\"https://service-manual.nhs.uk/design-system/guides/updating-to-v10\">upgrading to version 10 guide</a> to make any required changes to your code.</p>"
```